lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
About brute-force. Just think about this:
```bash
$ curl -o /dev/null -s -w {time_connect}:%{time_starttransfer}:%{time_total}\\n  https://evolve.cipherhealth.com/home
{time_connect}:1,527:1,527
```
Even with three servers with multithreading 1.5 seconds responce with be a nightmare for hacker. Especially if session dies after two days.